src/Repository/CommissionableAssetRepository.php

```diff
@@ -26,6 +26,20 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, CommissionableAsset::class);
     }
 
+    /**
+     * @return CommissionableAsset[]
+     */
+    public function search(string $query): array
+    {
+        $words = explode(' ', $query);
+        $qb = $this->createQueryBuilder('ca')->innerJoin('ca.organization', 'o');
+        foreach ($words as $i => $word) {
+            $qb->andWhere("ca.name LIKE ?$i OR ca.contact LIKE ?$i OR o.name LIKE ?$i")->setParameter($i, "%$word%");
```

Comment:
this search is not case insensitive. You should use `ILIKE` and add proper LOWER indexes : https://api-platform.com/docs/core/performance/#search-filter

In migrations : 
```php
        $this->addSql('CREATE INDEX user_firstname_lower_idx ON users (LOWER(firstname))');
```